config/locales/en.yml

```
@@ -1764,11 +1764,12 @@ en:
   mailers:
     applicant:
       enrollment_verification:
-        body: To activate your %{school_name} account, and enroll to the %{course_name} course, please verify your email address. Clicking on this <strong>one-time link</strong> will verify your email-address and confirm that you wish to create an account at %{school_name}.
+        link_to: this link
```

```suggestion
        link_to: link
```
`This` won't be required as `body_html` has it
```
body_html: To activate your %{school_name} account, and enroll to the %{course_name} course, please verify your email address. Clicking on this %{link_to} will verify your email-address and confirm that you wish to create an account at %{school_name}.
```

---

It's perhaps bad naming of the keys that is causing the confusion here. Will rename it as a fix